apps/babanews/pages/a/index.tsx

```diff
@@ -49,24 +50,31 @@ export default function ArticleList() {
     return 1
   }, [data?.articles.totalCount])
 
+  const canonicalUrl = '/a'
```

Comment:
Canonical urls should be absolute ideally (but I understand why they aren't here due to SSR) but can be relative. So I guess it's fine to keep it like this.